Changelog.md

```diff
@@ -1,8 +1,9 @@
-## 4.3.1 - Unreleased
+## 4.4.0 - Unreleased
 
 ### Changed
 
--   Notarize with stricter entitlements for macOS
+-   Notarize with stricter entitlements for macOS.
```

Comment:
What does this mean? I don't recall any notaries or power of attorneys being involved with nRF Connect for Desktop development...